include/boost/geometry/algorithms/detail/calculate_sum.hpp

```diff
@@ -30,7 +34,7 @@ class calculate_polygon_sum
     template <typename ReturnType, typename Policy, typename Rings, typename Strategy>
     static inline ReturnType sum_interior_rings(Rings const& rings, Strategy const& strategy)
     {
-        ReturnType sum = ReturnType();
+        ReturnType sum = ReturnType(0);
```

Comment:
Yes, indeed I remember the workaround.
However, we the construction (as it was) a lot in the whole Boost.Geometry code. It was suggested by Bruno, and the style (call the template argument as a constructor, even for value types which would otherwise be uninitialized) is made for templates (I see that I did write this before).

It makes no sense to change it occasionally, in one location. If you want to change it, we should change it everywhere. But I'm not so sure about that.